app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
Ah, you're saying that in research notes, it's a unique slug, but in maps,
it's not. I'm not sure what to do about that.

I'm a bit stumped... so just to throw out a possibility, could you (just
for maps, as a special case) string replace the "/" char with another char,
like "*" or something, so that the URL forms a unique friendly_id, or
perhaps have a custom route just for maps, with `find( params[:slug] + '/'
+ params[:date])` , just for this one case?

On Mon, Jul 25, 2016 at 1:37 PM, Ananya Maiti <notifications@github.com>
wrote:

> In app/controllers/map_controller.rb
> <https://github.com/publiclab/plots2/pull/600#discussion_r72108518>:
>
> > @@ -16,7 +16,10 @@ def index
> >    end
> >
> >    def show
> > -    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
> > +    @node = DrupalNode.find params[:name]
>
> Yeah we can always use an url like /map/NewYork-map-07-22-2016/07-22-2016
> for maps (where NewYork-map-07-22-2016 is the slug) and something similar
> to that in notes. Only thing is that it looks like overuse of the date in
> the slugs.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/600/files/c968e35c5fa7b3232e5e5b2ca0bc9afa658586cd#r72108518>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJxMoNcyjpTSf1XmJXDPdsXoOoqOFks5qZPRCgaJpZM4JDL2B>
> .
>